### DISCUSSION TITLE
manylinux2014 EOL

### DISCUSSION THREAD

**Discussion by thomasfulton** [pypa/manylinux#1648] [Q&A] [answered]
Centos 7 reached EOL on June 30. They took down mirrorlist.centos.org on that date so yum installs no longer work. Might be time to officially EOL manylinux2014

**Comment by mayeut** [comment]
The mirrors have been update in https://github.com/pypa/manylinux/pull/1628
The images will still be updated for the time being, probably for at least one year depending on statistics.

**Comment by mayeut** [comment, marked as answer]
EOL for manylinux2014 has been set to 2027-05-04 (https://github.com/pypa/manylinux/issues/1925)
